cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/RenameController.java

```diff
@@ -0,0 +1,87 @@
+package cloud.foundry.cli.services;
+
+import static picocli.CommandLine.*;
+import static picocli.CommandLine.usage;
+
+import cloud.foundry.cli.crosscutting.exceptions.UpdateException;
+import cloud.foundry.cli.crosscutting.logging.Log;
+import cloud.foundry.cli.crosscutting.mapping.CfOperationsCreator;
+import cloud.foundry.cli.operations.ApplicationsOperations;
+import cloud.foundry.cli.operations.ServicesOperations;
+import org.cloudfoundry.operations.DefaultCloudFoundryOperations;
+import reactor.core.publisher.Mono;
+
+import java.util.concurrent.Callable;
+
+/**
+ * This class realizes the functionality that is needed for the rename commands.
+ * You can rename applications or services.
+ */
+@Command(name = "rename", header = "%n@|green Rename an application or a service.|@",
+        subcommands = {RenameController.RenameApplicationCommand.class,
+        RenameController.RenameServiceCommand.class})
+public class RenameController implements Callable<Integer> {
+
+    @Override
+    public Integer call() throws Exception {
+        usage(this, System.out);
+        return 0;
+    }
+
+    @Command(name = "application", description = "Rename an application.")
+    static class RenameApplicationCommand implements Callable<Integer> {
+
+        private static final Log log = Log.getLog(RenameApplicationCommand.class);
+
+        @Mixin
+        LoginCommandOptions loginOptions;
+
+        @Parameters(index = "0", description = "The current name of the app")
+        private String currentName;
+
+        @Parameters(index = "1", description = "The new name of the app")
+        private String newName;
+
+        @Override
+        public Integer call() throws Exception {
+            log.info("Renaming application...");
+            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
+            ApplicationsOperations applicationOperations = new ApplicationsOperations(cfOperations);
```

Comment:
"*Operations" should be accessed by a corresponding Logic class.

Reply:
I think it is an overkill in this place.

Reply:
Why? 
We did it also in other Controllers and want keep the stringent, like it is defined in the architecture.

Reply:
Because it is only 2 lines of code. But i get your point. Let's do it to be consistent.